.luacheckrc

```diff
@@ -1,14 +1,21 @@
 unused_args = false
 allow_defined_top = true
+max_line_length = 185
+
+exclude_files = {".luacheckrc"}
```

Comment:
This isn't needed, `.luacheckrc` is not a `.lua` file, so it's already ignored :P